main.tf

```diff
@@ -302,6 +302,33 @@ resource "kubernetes_limit_range_v1" "this" {
   }
 }
 
+################################################################################
+# K8s Admin ClusterRole Binding
+################################################################################
+
+resource "kubernetes_cluster_role_binding_v1" "admin" {
+  count = var.create_cluster_role && var.enable_admin ? 1 : 0
+
+  metadata {
+    name        = coalesce(var.role_name, var.name)
+    annotations = var.annotations
+    labels      = var.labels
+  }
+
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "admin"
+  }
+
+  subject {
+    kind      = "Group"
+    name      = var.name
```

Comment:
I think this has a different meaning now - I'm not sure that `var.name` makes sense here

Reply:
I'll provide new variables for this resource.